.github/workflows/_unit-tests.yaml

```diff
@@ -8,6 +8,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
       - uses: ./.github/actions/rebase
```

Comment:
Won't the `rebase` action break this fix anyway? It switches to `github.sha` commit